.github/workflows/metadata-governance.yml

```
@@ -80,3 +83,46 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
+
+      # ── Native issue type sync ────────────────────────────────────────────────
+      # Sets the GitHub native issue type (Bug / Feature / Task / etc.) directly
+      # on the issue using the type:* label → project_field_mappings.Type mapping
+      # in .github/issue-fields.yml. Requires a GitHub App token with
+      # issues:write and project scope (supplied via LS_APP_PRIVATE_KEY secret).
+      - name: Create GitHub App token for native type sync
+        id: app-token
+        if: |
+          github.event_name == 'issues' &&
+          vars.LS_APP_ID != '' &&
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)